paddle/fluid/platform/profiler.cc

```diff
@@ -112,6 +117,7 @@ Event::Event(EventType type, std::string name, uint32_t thread_id,
     auto stream = cuda_dev_ctx->stream();
     PADDLE_ENFORCE(cudaEventRecord(event_, stream));
   }
+#endif
```

Comment:
We should only support `cupit` to profile program, because the installing of `cupit`  is easy, i.e `sudo apt-get install cuda-command-line-tools-9-0`  for CUDA 9.0.

Reply:
`cudaEvent` is completely removed from the profiler.